src/views/summary/Summary.test.tsx

```diff
@@ -55,7 +55,7 @@ describe("Summary", () => {
     fetch.mockReset();
   });
 
-  test("Should show questions and answers", async () => {
+  test.skip("Should show questions and answers", async () => {
```

Comment:
Hvorfor er disse testene skippet?

Reply:
Testen feilet og jeg var litt usikker på hvordan det kan fikses, skippet for å kunne få teste i dev. Men skal fikse det før det merges.

Reply:
Den er god 😊

Reply:
@hanneolsen  fixed. Jest og react hooks fungerer litt dårlig sammen. 

Fikk feilen `ref.current.scrollIntoView` is not a function. Måtte legge til denne for at testen skal kjøre gjennom. 

`Element.prototype.scrollIntoView = jest.fn();`
  https://github.com/jsdom/jsdom/issues/1695#issuecomment-449931788